Parser/asdl_c.py

```diff
@@ -638,9 +638,13 @@ def visitModule(self, mod):
 ast_dealloc(AST_object *self)
```

Comment:
While you're in here, could you fix the typo introduced here: https://github.com/python/cpython/blob/d057b896f97e6d7447b9bf9246770c41cf205299/Python/Python-ast.c#L1236-L1237

(`Py_tp_free` is duplicated, once incorrectly set to `PyType_GenericNew`)